app/Config/Routes.php

```diff
@@ -14,3 +16,6 @@
 $routes->post('account/login', [Account::class, 'login']);
 $routes->post('account/logout', [Account::class, 'logout']);
 $routes->get('account', [Account::class, 'get'], ['filter' => AuthFilter::class]);
+$routes->get('images/(:segment)', [Image::class, 'get']);
+$routes->get('events/(:num)', [Event::class, 'get']);
```

Comment:
A route for an hole event without the `visible_from` date will be implemented with the backend routes?

Reply:
Yes, for the backend there will be specialized routes that also require authentication. I'd like to add those in a separate PR. I will track this here: https://github.com/TechStreamConference/website-backend/issues/21